repositories/SectionRepository.test.js

```diff
@@ -114,6 +114,19 @@ describe("SectionRepository", () => {
     expect(position).toEqual("0");
   });
 
+  it("can get section count ", async () => {
+    const {
+      questionnaire: { id: questionnaireId }
+    } = await setup();
+
+    await SectionRepository.insert(
+      buildSection({ questionnaireId: questionnaireId })
+    );
+
+    const count = await SectionRepository.getSectionCount(questionnaireId);
+    expect(count).toEqual("1");
```

Comment:
Why is 1 a string as opposed to a number?

Reply:
From Knex docs

```
count — .count(column|columns|raw)

Performs a count on the specified column or array of columns (note that some drivers do not support multiple columns). Also accepts raw expressions. Note that in *Postgres*, count returns a bigint type which will be a String and not a Number (more info).
```

Reply:
Postgres returns String as count value